poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFRun.java

```diff
@@ -1564,6 +1564,9 @@ private void _getText(XmlObject o, StringBuilder text) {
             final Node node = o.getDomNode();
             if (XSSFRelation.NS_WORDPROCESSINGML.equals(node.getNamespaceURI())) {
                 switch (node.getLocalName()) {
+                    case "noBreakHyphen":
+                        text.append('‑');
```

Comment:
is this a simple minus character or is the Unicode Non-Breaking-Hyphen char (https://www.compart.com/en/unicode/U+2011)?

Reply:
Due to the inability to directly copy the ‑ character from Word, I chose Unicode Non-Breaking-Hyphen char(/u2011), which has the same meaning, as the replacement for w:noBreakHyphen